WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsViewModel.swift

```diff
@@ -105,6 +105,13 @@ final class PaymentMethodsViewModel: ObservableObject {
 
     private let cardPresentPaymentsConfiguration: CardPresentPaymentsConfiguration
 
+    private let cardUpsellCampaign =  UpsellCardReadersCampaign(source: .paymentMethods)
+
+    var cardUpsellAnnouncementViewModel: FeatureAnnouncementCardViewModel {
+        .init(analytics: analytics,
+              configuration: cardUpsellCampaign.configuration)
+    }
+
```

Comment:
Should we rename the variables starting with `cardUpsell...` to `upsellCard...` for consistency with the campaign name?

Reply:
Will do this in a future PRs to save on rebasing and re-running tests in CI, since it's a simple one.